htdocs/modulebuilder/index.php

```diff
@@ -1367,12 +1365,13 @@
 			}
 		}
 
+
 		if (!$error) {
 			foreach ($filetogenerate as $srcfile => $destfile) {
 				$result = dol_copy($srcdir.'/'.$srcfile, $destdir.'/'.$destfile, $newmask, 0);
 				if ($result <= 0) {
 					if ($result < 0) {
-						$warning++;
+						$error++;
```

Comment:
It seems you didn't merge your change with the main branch correctly. Some changes done by other PR are lost like this one.